spec/fixtures/generate_external_certs.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+
+# This script re-generates all of the keys and certs within the external_certs directory.
+# The files there are used by the spec/features/settings/external_cert_feature_spec test
+
+function init() {
+  ROOT_DIR=/tmp/ca_gen
```

Comment:
Should this perhaps use `${TMPDIR:-/tmp}` to allow a test environment which has exported `$TMPDIR` to generate wherever it's using for temporary files?  Or even better, let mktemp deal with that for you, and generate a guaranteed unique directory name at the same time.
```suggestion
  ROOT_DIR=$( mktemp -d )
```

Reply:
As you might imagine this whole script is meant to be able to be used in a generic fashion but is not yet to that point. When it is made into a fully standalone script accepting a DSL for the list of roots and certs etc, then it will do more of what you are suggesting. For now your mktemp suggestion is great. Adding that. Thanks.